rpm/mosh.spec

```diff
@@ -27,12 +28,17 @@ Mosh is a remote terminal application that supports:
 
 
 %prep
-%setup -q
+# Validate we're using the tarball we expect.
+SHA256="e7f69404816f1ca5b44762d010e406a56d18e133d31efd0783970de8d6c80ed4"
+test $(sha256sum ../SOURCES/%{name}-%{version}.tar.gz |cut -d " " -f 1) == "$SHA256"
```

Comment:
This violates the RPM abstraction layer that allows the location of source tarballs to be configured; it need not be `../SOURCES`.

I’ve never seen an RPM spec that tries to checksum a source tarball—is that standard practice somewhere?